metadata.yaml

```diff
@@ -27,3 +27,7 @@ provides:
     interface: cos_agent
   rpc-url:
     interface: rpc-url
+
+requires:
+  relay-rpc-url:
+    interface: rpc-url
```

Comment:
Hmm would it make sense for these two to have more aligned names, to show elsewhere in the code that they're different sides of the same interface?

E.g. `para-rpc-url` (used to provide from relays) and `relay-rpc-url` (used to require from paras)

Reply:
Well, the thing is that the provider might not only provide to parachain nodes. The `rpc-url` relation was already there to provide endpoint for `substrate-api-sidecar`. And both relaychain nodes and parachain nodes can provide that.

Reply:
Ah ok! Then leave it as is I guess :)